pkg/bitwarden/cli.go

```diff
@@ -142,3 +144,170 @@ func (c *cliClient) GetPassword(itemName string) ([]byte, error) {
 func (c *cliClient) Logout() ([]byte, error) {
 	return c.run("logout")
 }
+
+func createItem(itemTemplate string, targetItem *Item, c *cliClient) error {
+	// the bitwarden cli expects the item to be base64 encoded
+	encItem := b64.StdEncoding.EncodeToString([]byte(itemTemplate))
+	out, err := c.run("--session", c.session, "create", "item", encItem)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(out, targetItem)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func createAttachment(fileContents []byte, fileName string, itemID string, newAttachment *Attachment, c *cliClient) error {
+	filePath := "/tmp/" + fileName
```

Comment:
```suggestion
	filePath := filepath.Join(os.TempDir(), fileName)
```

Reply:
Can we have name clashes since we have a concurrency option?  We can use a random name if the file name is not important for the `create` command.  We also need to delete the temporary file.

Reply:
I think the file name is important when creating the attachment.. I can add a /tmp/itemID/fileName to avoid concurrency issues.
But I guess itemID will have illegal characters for a folder(I'll pass in item name as an argument to handle that)
Yes,  I'll close the file as well.

Reply:
I have now made these tempDir - so I think we wouldn't have concurrency issues if we were to make it parallel